package.json

```diff
@@ -32,6 +32,7 @@
     "mocha": "^3.0.0",
     "mocha-eslint": "^3.0.1",
     "quick-temp": "^0.1.3",
+    "sinon": "^1.17.7",
```

Comment:
I have a general hatred of `sinon`, and it seems that its essentially only being used to stub `console.log`.  

In ember-cli ecosystem we generally want all console output to go through our `ui` object so that things like the `--silent` option are respected. Typically, this is done by using `this.console` and having that default to `console` if not passed as an option. With that solution, we make ember-cli better *and* avoid `sinon`...

Example from `broccoli-lint-eslint`: [here](https://github.com/ember-cli/broccoli-lint-eslint/blob/master/lib/index.js#L119),[here](https://github.com/ember-cli/broccoli-lint-eslint/blob/master/lib/index.js#L205), and [here](https://github.com/ember-cli/ember-cli-eslint/blob/master/index.js#L67-L75)